packages/nx/src/command-line/release/changelog.ts

```diff
@@ -1446,3 +1487,18 @@ function versionPlanSemverReleaseTypeToChangelogType(bump: ReleaseType): {
       throw new Error(`Invalid semver bump type: ${bump}`);
   }
 }
+
+function extractPreId(version: string): string | undefined {
```

Comment:
Your casing is inconsistent with the option, let's change that please (for the same reason Miro wanted it changed and we said no), we need to commit either way